### ISSUE TITLE
sgx_ippcp.h under external/epid-sdk/ext/ipp is not up-to-date (?)

### ISSUE THREAD

**Issue by sbellem** [intel/linux-sgx#759]
Is this header file supposed to be in sync with the one that ships in the optmized libs that can be downloaded from https://01.org/intel-software-guard-extensions/downloads?

A quick inspection (with `diff`) shows that the header file for 2.15 differs than the one under `external/epid-sdk/ext/ipp`. Should they be in sync?

Related to #752.

**Comment by lzha101** [comment]
You are right. Seems we forgot to sync this file under the epid folder. Thanks for pointing out this. We will fix this although it doesn't impact the functionality.

**Comment by veehaitch** [comment]
Great, thanks! While you're at it, it would be great to also address #752 which also revolves around the `sgx_ippcp.h` header file.
